### DISCUSSION TITLE
[discussion] 색상 정보 관리를 위한 index.css에서 클래스로 관리 여부 의견

### DISCUSSION THREAD

**Discussion by July249** [beyondDevelops/AnimalTalk#28] [Q&A] [answered]
색상 정보를 관리의 용이성을 위해 .btn-active, .btn-inactive 등과 같이 ts Boolean에 따른 스타일 관리를 index.css에서 클래스로 진행하는 것이 어떠할까요?

Originally posted by @July249 in https://github.com/beyondDevelops/AnimalTalk/pull/24#discussion_r1047227315

**Comment by GreattitJY** [comment, marked as answer]
좋아요. 유지보수 측면에서 더 용이할 거 같습니다!
